src/redux/actions/types.js

```diff
@@ -1,3 +1,8 @@
 export const FETCH_EPISODES_BEGIN   = 'FETCH_EPISODES_BEGIN';
 export const FETCH_EPISODES_SUCCESS = 'FETCH_EPISODES_SUCCESS';
 export const FETCH_EPISODES_FAILURE = 'FETCH_EPISODES_FAILURE';
+
+export const SHOW_SELECTED = 'SHOW_SELECTED';
```

Comment:
I'd suggest to pay more attention about naming. If I will be new in this project what the 'SHOW_SELECTED' action tells me about what it does?